js/example.js

```diff
@@ -116,7 +116,11 @@ module.exports = function (styleguide, filePath) {
   // Returns all pairs who's keyname doesn't start with an '_'
   function removePrivateKeys (data) {
     return _.omit(data, function (value, key) {
-      return _.startsWith(key, '_')
+      if (Object(value) === value) {
+        removePrivateKeys(value)
+      }
+
+      return _.startsWith(key, '_') && !_.eq(key, '_view')
```

Comment:
What do you think about just using `(key !== '_view')` here to keep our usage of lodash to a minimum?